magento2-nginx/fpm-7.0/etc/confd/conf.d/magento_env.php.toml

```diff
@@ -0,0 +1,8 @@
+[template]
+src   = "magento/env.php.tmpl"
+dest  = "/app/app/etc/env.php"
```

Comment:
Might need to `mkdir -p /app/app/etc/` before templating runs, just in case. See https://github.com/continuouspipe/dockerfiles/blob/master/drupal8-apache/7.0/usr/local/share/container/baseimage-30.sh#L15-L19